Update upgrade (#8985)
fishtown-analytics/docs.getdbt.com: website/docs/guides/fusion-upgrade-prepare.md

:::

## Enable Fusion readiness features

> Before your team can view <Constant name="fusion" /> readiness status or access the upgrade assistant, an account admin must enable the <Constant name="fusion" /> readiness experience in your account settings. This lets admins and developers see which projects and jobs are eligible for <Constant name="fusion" />, identify blockers, and initiate the upgrade once preparation is complete.

Reviewer: Same comment here on the roll-out making this tricky to communicate

> The <Constant name="fusion" /> readiness panel in the <Constant name="dbt_platform" />, which shows each project's eligibility status and blockers, is being rolled out in phases. If it hasn't been automatically turned on for your account yet, an account admin can manually enable it. This lets admins and developers see which projects and jobs are eligible for <Constant name="fusion" />, identify blockers, and initiate the upgrade once preparation is complete.

Reviewer 2: ```suggestion
The <Constant name="fusion" /> readiness panel in the <Constant name="dbt_platform" /> and shows each project's eligibility status and blockers, is being rolled out in phases. If it hasn't been automatically enabled for your account yet, an [account admin](/docs/cloud/manage-access/enterprise-permissions#account-admin) can manually enable it. This lets admins and developers see which projects and jobs are eligible for <Constant name="fusion" />, identify blockers, and initiate the upgrade once preparation is complete.
```

### Step 1: Enable the readiness toggle

This step requires account admin access in <Constant name="dbt_platform" />:

1. Click your account name in the left sidebar and select **Account settings**.
2. Navigate to the **Account** screen and click **Edit**.

> 3. Scroll to the **Settings** section and click the checkbox next to **Enable Fusion readiness & upgrade features**.

Reviewer 2: ```suggestion
3. Scroll to the **Settings** section and select the checkbox next to **Enable Fusion readiness & upgrade features**.
```

Reviewer 2: Hiya Matt, 

Just amending this so that it aligns with the style guide wording for UI elements

4. Click **Save**.

Once enabled:
- All admins and developers can see each project's <Constant name="fusion" /> readiness status, including which jobs are eligible or ineligible for <Constant name="fusion" /> and why.
- Admins can initiate the <Constant name="fusion" /> upgrade from development environments, environment settings, and job settings (subject to existing permissions).

### Step 2: Restrict upgrade access (optional)

By default, any user who can see the upgrade assistant can use it to initiate an upgrade. To limit upgrade execution to designated users, you can enable an additional access control toggle.

> :::note Enterprise only

Reviewer 2: Hiya Matt

Even though the callout has info advising it's for both enterprise and enterprise+ customers, I think this should be explicitly clear in the header to ie something like "Enterprise and Enterprise+ only" or "Enterprise accounts only"

> :::note Enterprise only

Author: ```suggestion
:::note Enterprise tier accounts only
```

The **Enable restricted Fusion upgrade permissions** toggle is only available to Enterprise/Enterprise+ accounts that have been granted this entitlement. Contact your account manager if you need this capability.

:::

> 1. In the same **Account** settings screen, click the checkbox next to **Enable restricted Fusion upgrade permissions**.

Reviewer 2: ```suggestion
1. In the same **Account** settings screen, select the checkbox next to **Enable restricted Fusion upgrade permissions**.
```

2. Click **Save**.

> When enabled, only users assigned the `Fusion admin` permission set (scoped to specific projects) can execute the upgrade. For instructions on assigning this permission, see [Assign upgrade access](/guides/upgrade-to-fusion?step=3#assign-upgrade-access-optional) in Part 2 of this guide.

Reviewer 2: ```suggestion
When enabled, only users assigned the [`Fusion admin`](/docs/cloud/manage-access/enterprise-permissions#fusion-admin) permission set (scoped to specific projects) can execute the upgrade. For instructions on assigning this permission, refer to [Assign upgrade access](/guides/upgrade-to-fusion?step=3#assign-upgrade-access-optional) in Part 2 of this guide.
```

### The Fusion readiness panel

With the readiness experience enabled, you can monitor your project's eligibility as you work through the preparation steps below. The panel shows which jobs are eligible or ineligible for <Constant name="fusion" /> and why.

<Lightbox src="/img/fusion/fusion-readiness.png" width="60%" title="The Fusion readiness checklist"/>

Common ineligibility reasons include:

> - Environment(s) not on the **Latest** release track

Reviewer 2: ```suggestion
- Environment(s) not on the **Latest** [release track](/docs/dbt-versions/cloud-release-tracks#which-release-tracks-are-available]
```

- Not using a [supported data platform](/docs/fusion/supported-features?version=2.0#requirements)

> - Project doesn't have at least one succesful job run

Reviewer 2: ```suggestion
- Project doesn't have at least one successful job run
```
